drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -880,7 +908,17 @@ function DragAndDropBlock(runtime, element, configuration) {
                 }
             }
         }
-    }
+    };
+
+    var initializeSlider = function() {
+        if (pageLoaded) {
+            itemSlider = $root.find('.item-bank').bxSlider({
+                pager: false,
+                touchEnabled: false,
+                infiniteLoop: false
+            });
```

Comment:
Looks like you can either style the "prev" and "next" text control links pretty heavily (which seems painful), or provide custom prev/next controls and pass them as [`nextSelector`/`prevSelector` options](https://bxslider.com/options/).

This [stackoverflow post](https://stackoverflow.com/a/16065654) also shows an approach that hooks up the click handlers, but I don't think you need to do that if you use the selectors.

Reply:
We will discuss this in styling of this slider.